### PR TITLE
MTP-1879 designjusteringer for lansering

### PR DESCRIPTION
**Pull request by petterkoding**
increase spacing between fieldset and error border

### CODE REVIEW

File: mt-kit/core/css/src/_theme.scss [hunk 1]
```diff
@@ -11,6 +12,7 @@ $focus-box-shadow-on-white: 0 0 0 0.125rem var(--color-mt-white),
 $focus-box-shadow-no-outline: 0 0 0 0.25rem var(--mt-button-primary-bg-focus);
 $error-box-shadow: 0 0 0 0.5rem var(--color-mt-background), 0 0 0 0.7rem var(--color-error);
 $error-box-shadow-small: 0 0 0 0.05rem var(--color-mt-background), 0 0 0 0.2rem var(--color-error);
+$error-box-shadow-medium: 0 0 0 1.05rem var(--color-mt-background), 0 0 0 1.2rem var(--color-error);
```
Comment (lha047) [inline mt-kit/core/css/src/_theme.scss:15]:
```suggestion
$error-box-shadow-medium: 0 0 0 var(--spacer-xx-small) var(--color-mt-background),
  0 0 0 calc(var(--spacer-xx-small) + $border-width-large) var(--color-error);
```
Suggest using variables for this

**Review by lha047 — CHANGES_REQUESTED** [review]
Use variables for calculating spacing in box shaddow

File: mt-kit/core/css/src/_theme.scss (hunk 1 above)
Comment (petterkoding) [reply mt-kit/core/css/src/_theme.scss:15]: @lha047  I dont think using the same values will work, as the borders are overlapping. The error border must be bigger to show underneath the "masking" border.

Making the second border  `--spacer-x-small` will make it look too big imo

Comment (lha047) [reply mt-kit/core/css/src/_theme.scss:15]: Thats why you calcualte the red border

Comment (petterkoding) [reply mt-kit/core/css/src/_theme.scss:15]: Haha, my tired eyes just didnt see it! Sorry

**Review by lha047 — APPROVED** [review]
Nice :D

**Pull request merged**